web/styles/zulip.css

```diff
@@ -666,6 +666,103 @@ body.has-overlay-scrollbar {
     }
 }
 
+.upload-overlay-container {
+    visibility: hidden;
+    position: fixed;
+    display: flex;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1em;
+    /* Semi-transparent background with a blur effect */
+    background: hsl(238deg 35% 15% / 65%);
+    backdrop-filter: blur(8px);
+    transition:
+        transform 400ms,
+        opacity 400ms;
+    opacity: 0;
+    z-index: -1;
+    pointer-events: none;
+}
+
+.upload-overlay-visible .upload-overlay-container {
+    visibility: visible;
+    z-index: var(--upload-overlay-z-index);
+    opacity: 1;
+    pointer-events: all;
+}
+
+.upload-overlay-svg-container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+}
+
+.upload-overlay-svg {
+    /* We have left 80px from both sides of the display.
+    For more information regarding values, refer
+    <https://www.figma.com/file/jbNOiBWvbtLuHaiTj4CW0G/Zulip-Web-App?type=design&node-id=6229-148055&mode=dev> */
+    width: calc(100% - 5em); /* 80px at 16px/1em */
+    height: calc(100% - 5em); /* 80px at 16px/1em */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @media (width <= $sm_min) {
+        width: calc(100% - 1.25em); /* 20px at 16px/1em */
+        height: calc(100% - 1.25em);
+    }
+}
+
+.upload-overlay-svg-rect {
+    /* The border SVG is a square with 3px border.
+    For more information regarding values, refer
+    <https://www.figma.com/file/jbNOiBWvbtLuHaiTj4CW0G/Zulip-Web-App?type=design&node-id=6229-148055&mode=dev> */
+    width: calc(100% - 3px);
+    height: calc(100% - 3px);
+}
+
+.upload-overlay-content {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 16px;
+    text-align: center;
+    color: hsl(0deg 100% 100%);
+    z-index: 10001;
+    min-width: 280px;
+    max-width: 610px;
```

Comment:
Why are these numbers in pixels? We should be using `em` for all new values. Also, I don't think this CSS is the cause, but if you do a window size that's kinda phone-shaped (tall, narrow), it doesn't look good  -- the rectangle doesn't go to the edge properly. I'm not sure what the cause is.

Here's a sample pre-drag shape that doesn't work.

<img width="977" height="1483" alt="Image" src="https://github.com/user-attachments/assets/c710ed62-876c-4e7f-8ea4-fd198ac0143c" />